q2_sample_classifier/_transformer.py

```diff
@@ -102,7 +102,7 @@ def _9(ff: ImportanceFormat) -> (qiime2.Metadata):
 def _10(data: pd.DataFrame) -> (ProbabilitiesFormat):
     ff = ProbabilitiesFormat()
     with ff.open() as fh:
-        data.to_csv(fh, sep='\t', header=True, na_rep=np.nan)
+        data.to_csv(fh, sep='\t', na_rep=np.nan, header=True)
```

Comment:
Nothing actually changed (functionally) on this line, right?

Reply:
correct, nothing changed functionally. I was wondering about that.

Reply:
@David-Rod - one thing to keep in mind in the future is that now this line will show up on a `git blame` related to this PR, which is maybe not going to be very helpful for future devs reading this code. No worries, just something to keep in mind for the future.